ui/webui/test/check-progress

```diff
@@ -31,11 +31,19 @@ class TestInstallationProgress(anacondalib.VirtInstallMachineCase):
         i = Installer(b, self.machine)
 
         i.open()
+        drawer_open = False
 
         for page in i.steps[:-1]:
             # with the pages basically empty of common elements (as those are provided by the top-level installer widget)
             # we at least iterate over them to check this works as expected
             i.wait_current_page(page)
+            if drawer_open:
+                self.assertFalse(b.is_present(".pf-c-drawer__panel-main"))
```

Comment:
I think the common practice in our tests is to use the element ids via helper functions in test/helpers/*.py. For example check_ or click_ funcitons in test/helpers/storage.py. So maybe it is worth following it also here, like having check_drawer_open function somewhere?

Reply:
Added helper functions